package.json

```diff
@@ -88,5 +88,9 @@
   },
   "publishConfig": {
     "registry": "https://registry.npmjs.org"
+  },
+  "volta": {
```

Comment:
Hacks to make the tests JustWork™️ in my dev env. Kinda weird that we use pnpm for the project but `yarn install` in the tests...can fix later.